www/src/components/homepage/homepage-newsletter.js

```diff
@@ -11,100 +12,90 @@ import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
 const stripedBorderHeight = 1
 
-const Container = styled(`div`)`
-  background: ${p => p.theme.colors.newsletter.background};
-  border: 1px solid ${p => p.theme.colors.newsletter.border};
-  border-radius: ${p => p.theme.radii[2]};
-  display: flex;
-  flex-direction: column;
-  margin-bottom: ${p => p.theme.space[8]};
-  padding: calc(${p => p.theme.space[8]} * 1.2);
-  padding-bottom: calc(
-    ${props => rhythm(props.theme.space[8] * 1.2)} +
-      ${p => p.theme.space[stripedBorderHeight]}
-  );
-  position: relative;
+const containerStyles = {
+  bg: `newsletter.background`,
+  border: t => `1px solid ${t.colors.newsletter.border}`,
+  borderRadius: 2,
+  flexDirection: `column`,
+  mb: 8,
+  p: t => `calc(${t.space[8]} * 1.2)`,
+  pb: t =>
+    `calc(${rhythm(t.space[8] * 1.2)} + ${t.space[stripedBorderHeight]})`,
+  position: `relative`,
 
-  :after {
-    border-radius: 0 0 ${p => p.theme.radii[2]} ${p => p.theme.radii[2]};
-    background: ${p => p.theme.colors.newsletter.background}
+  ":after": {
+    borderRadius: t => `0 0 ${t.radii[2]} ${t.radii[2]}`,
+    background: t => `${t.colors.newsletter.background}
       repeating-linear-gradient(
         135deg,
-        ${p => p.theme.colors.newsletter.stripeColorA},
-        ${p => p.theme.colors.newsletter.stripeColorA} 20px,
+        ${t.colors.newsletter.stripeColorA},
+        ${t.colors.newsletter.stripeColorA} 20px,
         transparent 20px,
         transparent 40px,
-        ${p => p.theme.colors.newsletter.stripeColorB} 40px,
-        ${p => p.theme.colors.newsletter.stripeColorB} 60px,
+        ${t.colors.newsletter.stripeColorB} 40px,
+        ${t.colors.newsletter.stripeColorB} 60px,
         transparent 60px,
         transparent 80px
-      );
-    bottom: 0;
-    content: "";
-    height: ${p => p.theme.space[stripedBorderHeight]};
-    left: 0;
-    right: 0;
-    position: absolute;
-  }
+      )`,
+    bottom: 0,
+    content: `""`,
+    height: t => t.space[stripedBorderHeight],
+    left: 0,
+    right: 0,
+    position: `absolute`,
+  },
 
-  ${mediaQueries.lg} {
-    flex-direction: row;
-    justify-content: space-between;
+  [mediaQueries.lg]: {
+    flexDirection: `row`,
+    justifyContent: `space-between`,
 
-    > * {
-      flex-basis: 50%;
-    }
-  }
-`
+    "> *": {
+      flexBasis: `50%`,
+    },
+  },
+}
 
-const Ornament = styled(`span`)`
-  left: -${p => p.theme.space[1]};
-  position: absolute;
-  top: -${p => p.theme.space[2]};
-`
+const ornamentStyles = {
+  left: -1,
+  position: `absolute`,
+  top: -2,
+}
```

Comment:
I'd make this a component and put the icon directly in it.

```suggestion
function Ornament() {
  return <span sx={{
  left: -1,
  position: `absolute`,
  top: -2,
}}>
  <NewsletterOrnament />
</span>
}
```

Alternately, I think the svg can be styled directly so you might just be able to do:

```jsx
<NewsletterOrnament sx={{
  left: -1,
  position: `absolute`,
  top: -2,
}} />

Reply:
also: do negatives work with the theme-ui sx properties?

Reply:
> also: do negatives work with the theme-ui sx properties?

Yep, they def. do work! Checked when refactoring https://github.com/gatsbyjs/gatsby/pull/25333/files#diff-b8226183ef427c89d128d6d47d8c31dc + co. Can't fully remember but I think that didn't work "before".

(… got curious:)

* https://github.com/system-ui/theme-ui/search?q=negative&unscoped_q=negative
* also mentioned in the docs now: https://theme-ui.com/sx-prop/#margin-and-padding  
  > Margin and padding are treated as first-class citizens in Theme UI, using values defined in the theme.space scale, and include an optional shorthand syntax for using negative space in your application.

Reply:
> I think the svg can be styled directly

Went for that ✌️